guides/common/modules/proc_importing-ansible-variables-from-yaml.adoc

```diff
@@ -0,0 +1,16 @@
+[id="importing-Ansible-variables-from-yaml-{context}"]
+= Importing Ansible variables from YAML files
+
+You can import Ansible variables from a `.yaml` file to {Project} within the {ProjectWebUI}.
+
+.Prerequisites
+* You must have at least one Ansible role imported.
+For more information, see xref:Importing_Ansible_Roles_and_Variables_{context}[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Configure* > *Ansible* > *Variables*.
+. Click *Import from YAML-File* in the toolbar.
++
+A wizard opens to show you the next steps to import/override Ansible variables.
+Ensure variables are unique per role.
```

Comment:
```suggestion
Ensure that variables are unique per Ansible role.
```

What happens if they're not?

Reply:
In case a variable is present > 1 times in the UI, i.e.:

- var0 <--
- var1
- var2
- var0 <--


 the user is not allowed to continue, as that would make the variable ambiguous.
 If a variable is present that has been imported earlier, the user may proceed, but must understand that the variable's type and default value will be overridden.
 Appropriate warnings are shown in both cases.